stream_alert_cli/package.py

```diff
@@ -220,7 +220,7 @@ def _upload(self, package_path):
                 return False
 
             package_fh.close()
-            LOGGER_CLI.debug('Uploaded %s to S3', package_name)
+            LOGGER_CLI.info('Uploaded %s to S3', package_name)
```

Comment:
Why make this `info`?

Reply:
It's to reduce the amount of messages printed, it was way too verbose before in info mode